component/vshn_postgres.jsonnet

```diff
@@ -122,13 +184,24 @@ local composition =
                                   [
                                     {
                                       sgObjectStorage: '',
-                                      cronSchedule: '',
+                                      cronSchedule: '0 22 * * *',
```

Comment:
Is this change intended? It looks like it's unrelated to the TLS configuration?

Reply:
this fixes issues with `default` in apis/vshn/v1/dbaas_vshn_postgresql.go because it's not set correctly, so I moved it here, it's harmless change which actually double ensure that correct schedule is applied to backup job

Reply:
I'd probably put it in a separate PR then, as it's not related to TLS at all. So that things are clean.